### PR TITLE
Apply spell/style check to top level markdown files

### PR DESCRIPTION
**Pull request by tillrohrmann**
[Spell/style check examples.md](https://github.com/restatedev/documentation/commit/5a445b77af28d1f8e106b0b2e1f0e26b0845743f)

[Spell/style check overview.md](https://github.com/restatedev/documentation/commit/de5aef2c52e74fa5d60c03e937b2cbbb1cca43c6)

[Spell/style check quickstart.mdx](https://github.com/restatedev/documentation/commit/66e68ab74498c46652198fbf1bda589c8ab32dea)

[Spell/style check tour.mdx](https://github.com/restatedev/documentation/commit/13cee4fff1d62593941d4fa02a8f57bdb2ad5d12)

This PR fixes https://github.com/restatedev/documentation/issues/142.

### CODE REVIEW

**Review by gvdongen — APPROVED** [review]
Thanks Till! LGTM :+1:

File: docs/tour.mdx [hunk 1]
```diff
@@ -250,11 +251,12 @@ The keys remain locked and the system can't process any more requests.
 ## Suspendable request-response calls
 :::note Implement it yourself or follow along by looking at the code under `src/part1`.
 :::
+
 One of the key parts of distributed applications is service-to-service communication.
 Restate makes sure that service-to-service communication is durable.
 Messages never get lost.
 
-Let's begin with request-response calls, in which one service calls another service and waits for the response.
+You begin with request-response calls, in which one service calls another service and waits for the response.
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply docs/tour.mdx:259]: I'll replace it with "The tour begins".

File: .github/styles/Google/Acronyms.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+extends: conditional
+message: "Spell out '%s', if it's unfamiliar to the audience."
+link: 'https://developers.google.com/style/abbreviations'
+level: suggestion
+ignorecase: false
+# Ensures that the existence of 'first' implies the existence of 'second'.
+first: '\b([A-Z]{3,5})\b'
+second: '(?:\b[A-Z][a-z]+ )+\(([A-Z]{3,5})\)'
+# ... with the exception of these:
+exceptions:
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply .github/styles/Google/Acronyms.yml:10]: Done

File: docs/tour.mdx [hunk 3]
```diff
@@ -969,18 +969,18 @@ This will tie the final parts together.
 :::
 
 Restate's replay mechanism makes applications resilient to failures.
-But it also requires your code to be deterministic.
+It requires your code to be deterministic.
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply docs/tour.mdx:972]: What about "It only requires"?

**Comment by tillrohrmann** [comment]
Thanks a lot for your review @gvdongen. I've addressed your comments. Merging this PR now.

**Pull request merged**